### DISCUSSION TITLE
GitHub student status validation system is probably broken

### DISCUSSION THREAD

**Discussion by KrisITB** [community/community#51873] [GitHub Education]
### Select Topic Area

Bug

### Body

Could someone please look into GitHub validation of student account processes? It seems that I'm one of many who are currently failing validation despite providing everything as requested by GitHub. I was trying to make a ticket but GitHub support bot clearly isn't prepared to handle such problems.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @KrisITB ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by randyb100** [comment]
I'm going through the same thing and the webcam function is broken. It will only let me take photos with the selfie cam. I took a photo of my portal, a letter from the National Student Clearinghouse verifying my current enrollment, and of a financial aid statement that includes my name, address, and student ID number. None of it was accepted. I'm using my student email address. I don't know how to validate that I'm a remote learner since none of the documents say that, and I don't know of any college that would say that on their documents. My university is almost solely online.

I also did some research and their Support page says that they DO NOT reply to student requests and WILL NOT individually review new accounts. What a shocker. It's like they don't want people to get it.

**Comment by samus-aran** [comment]
Hi @KrisITB & @randyb100 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient.

If you need help with applying please check out our [Apply to GitHub Global Campus for Students](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student) as well as our [Why wasn't my application to Global Campus for students approved?](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved). We aim to assist our students by providing rejection reasons throughout the application process so that you can take steps to rectify and re-apply. But we also understand that there are times where you need a little more guidance and we're here to nudge you in the right direction.

Let's see how we can assist ⭐ 

Here are some following tips regarding to commonly received rejection reasons, see if they apply to you:

1. The application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

2. For remote student applicants: please ensure that your evidence indicates that you are a remote student. If possible, we encourage students to apply whilst on campus. This will assist us in confirming your student status. If you are not able to be on campus we do accept evidence provided by institutions stating that they provide online courses. Reach out to your institution and request for a description of how your course will be delivered and submit it as part of your evidence.
